ime/app/src/main/java/com/anysoftkeyboard/keyboards/KeyboardSupport.java

```diff
@@ -71,17 +71,17 @@ public static int getKeyHeightFromHeightCode(
             KeyboardDimens keyboardDimens, int heightCode, float heightFactor) {
         int height;
         switch (heightCode) {
-            case 0:
-                height = 0;
+            case -1:
+                height = keyboardDimens.getNormalKeyHeight();
                 break;
             case -2:
                 height = keyboardDimens.getSmallKeyHeight();
                 break;
             case -3:
                 height = keyboardDimens.getLargeKeyHeight();
                 break;
-            default: // -1
-                height = keyboardDimens.getNormalKeyHeight();
+            default:
```

Comment:
Previously, if the value of `heightCode` was invalid, we would show the normal-key-height. Now, let's say `-7` comes in, what will happen?

Reply:
Good catch. I'll fix this.